### DISCUSSION TITLE
Exploring the Creative Side of Coding: First Week with Python Turtle 20.12.2023

### DISCUSSION THREAD

**Discussion by JamesCodingSpace** [JamesCodingSpace/Main#3] [General]
This weeks start was super exciting for me because I started learning about coding using Python [Turtle](https://github.com/JamesCodingSpace/Main/blob/codespace-urban-carnival-4jjvwx5vq6p6cgg/notebooks/TurtleKreis.py) graphics. It's like drawing pictures on a computer using math, and it's really cool!

Monday: Getting Started
Our teacher taught us how to use Python to make pictures. We talked about how to express ideas visually with code. Then, with a blank canvas, I was ready to start making my own creations.

Tuesday :Drawing Simple Shapes
Using Turtle was fun because it let me move a pretend "turtle" on the screen to draw lines. I started with easy shapes we learned in class, like squares and circles. It wasn't just about typing code; I could actually see my ideas showing up on the computer screen.

Wednesday: Making Fancy Designs
I learned about loops, which helped me create more complicated designs. Loops are like shortcuts that let you repeat and change patterns with just a little bit of code. This made it possible for me to make really cool and interesting pictures.

```
for _ in range(anzahl_dreiecke):
            dreieck_seite = 2 * radius * math.sin(math.radians(winkel / 2))
            for _ in range(3):
                pen.forward(dreieck_seite)
                pen.left(120)
            pen.left(winkel)
```

A Great Few Days
Looking back on these days, I'm surprised at how much I could do with Python Turtle. I went from drawing simple shapes to making more complex patterns. The best part was that I could control and play with my creations on the computer. Learning to code with Python Turtle was not only educational but also tons of fun!

I can't wait to keep learning more and exploring new things with coding. It's been a fantastic experience, and I'm excited about what comes next!

**Comment by SaskiaStier** [comment]
Hey, it looks like you learned a look about python and coding different shapes.
I think its amazing how we can draw different shapes just with some basic coding like for-loops and some print-commands.

**Comment by Vincent9956** [comment]
Keep it up ! I am at the same Level as you!

**Comment by nklshpf** [comment]
Nice Thread! Sounds very much like you had fun coding these last days. Keep on being motivated like this.

**Comment by alekey86** [comment]
Turtle is great! I also loved working with it!

**Comment by PhilippR84** [comment]
What a week! Keep it up!
